consensus/ibft/src/test/java/org/hyperledger/besu/consensus/ibft/validation/MessageValidatorTest.java

```diff
@@ -101,6 +106,12 @@ public void setup() {
             mockBftCtx,
             Optional.empty());
 
+    lenient()
+        .when(protocolSchedule.getByBlockNumberOrTimestamp(anyLong(), anyLong()))
```

Comment:
Thought that the MessageValidator was using ProtocolSchedule.getByBlockHeader now?